package.json

```diff
@@ -39,12 +39,13 @@
   "scripts": {
     "check": "gts check",
     "clean": "gts clean",
-    "compile": "tsc -p .",
+    "build": "tsc",
     "fix": "gts fix",
     "posttest": "npm run check",
+    "start": "node dist/server/server.js",
```

Comment:
In the production stage (see `Dockerfile`) we have only JS (not TS), so start the server using JS instead of `ts-node-dev`.